src/benchmarklib/benchmark_runner.cpp

```diff
@@ -503,6 +519,8 @@ cxxopts::Options BenchmarkRunner::get_basic_cli_options(const std::string& bench
     ("clients", "Specify how many items should run in parallel if the scheduler is active", cxxopts::value<uint32_t>()->default_value("1")) // NOLINT
     ("visualize", "Create a visualization image of one LQP and PQP for each query, do not properly run the benchmark", cxxopts::value<bool>()->default_value("false")) // NOLINT
     ("verify", "Verify each query by comparing it with the SQLite result", cxxopts::value<bool>()->default_value("false")) // NOLINT
+    ("memory_tracking", "Enables tracking of the temporary memory usage of certain operators using polymorphic allocators", cxxopts::value<bool>()->default_value("false")) // NOLINT
+    ("memory_tracking_output", "JSON file to output temporary memory tracking results to, don't specify for no output. Only applies if memory tracking is activated.", cxxopts::value<std::string>()->default_value("")) // NOLINT
```

Comment:
You could set the default directory to `/tmp`, if you want to have a proper default path.

Reply:
Would be an idea, but we decided to orientate on the other benchmark output.